lib/lunchbot_web/live/user_auth_live.ex

```diff
@@ -0,0 +1,19 @@
+defmodule LunchbotWeb.UserAuthLive do
+  import Phoenix.LiveView
+  alias Lunchbot.Accounts
+
+  # Helper for LiveView pages to get the current user.
+  # To use, declare "on_mount {LunchbotWeb.UserAuthLive, :user}"
+  # and reference in HEEX as "@current_user"
+  def on_mount(:user, _params, %{"user_token" => user_token} = _session, socket) do
+    socket =
+      socket
+      |> assign(:current_user, Accounts.get_user_by_session_token(user_token))
```

Comment:
nit: you can fit this all on one line without the pipe?

Reply:
eg: `socket = assign(socket, :current_user...)` will match the new value on right to the var socket.